src/scripts/update_funding_round.js

```diff
@@ -175,6 +200,6 @@ const main = async () => {
     }
 }
 
-module.exports = {addOCEANValueToEarmarks};
+module.exports = {completeEarstructuresValues};
```

Comment:
Not sure we need to export this function here

Reply:
Right, I'll remove it, thanks